docs/pages/ktexttruncator.vue

```diff
@@ -15,6 +15,35 @@
         </p>
       </DocsShow>
     </DocsPageSection>
+    <DocsPageSection
+      title="Usage"
+      anchor="#usage"
+    >
+      <p>
+        Use <code>KTextTruncator</code> to shorten long text while keeping it readable. It limits
+        the text to a specific number of lines and adds an ellipsis (<code>...</code>) when the text
+        overflows.
+      </p>
+
+      <p>By default, text truncates to one line if <code>maxLines</code> is not set.</p>
+
+      <h3>Customizing the Number of Lines</h3>
```

Comment:
```suggestion
      <h3>Customizing the number of lines</h3>
```

As per https://design-system.learningequality.org/writing#punctuation